src/components/Search/Searchbar.tsx

```diff
@@ -8,20 +9,19 @@ export default function Searchbar({
   defaultValue?: string
```

Comment:
question(non-blocking): not sure if `defaultValue` actually does anything here. Can this be removed or improved?